src/components/app-layout/app-layout.component.styles.ts

```diff
@@ -27,7 +27,8 @@ export const Content = styled.div`
   display: flex;
   flex-direction: row;
   flex: 0;
-  width: 80rem;
+  width: 100%;
```

Comment:
The home screen design is affected by this. Can you check?

Reply:
yeah, fixed-width issue.

Reply:
![image](https://user-images.githubusercontent.com/30016242/156215028-ab291a50-0b94-4ac4-bcd6-a21a81e5324d.png)